### PR TITLE
Swagger closure for Azure Migration Hub Operations

### PR DESCRIPTION
**Pull request by peetlotla**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/migrateprojects/resource-manager/readme.nodejs.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  package-name: azure-arm-migrate
+  package-version: 1.0.0-preview
+  output-folder: $(node-sdks-folder)/lib/services/migrateprojects
```
Comment (danschultequb) [inline specification/migrateprojects/resource-manager/readme.nodejs.md]: Please change this to `migrateprojectsManagement`.
```suggestion
  output-folder: $(node-sdks-folder)/lib/services/migrateprojectsManagement
```

File: specification/migrateprojects/resource-manager/readme.nodejs.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Node.js
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  package-name: azure-arm-migrate
```
Comment (danschultequb) [inline specification/migrateprojects/resource-manager/readme.nodejs.md]: Please reflect your service's complete name (`azure-arm-migrateprojects`).
```suggestion
  package-name: azure-arm-migrateprojects
```

**Comment by lmazuel** [comment]
@AutorestCI rebuild azure-sdk-for-java

**Comment by lmazuel** [comment]
@AutorestCI rebuild azure-sdk-for-java

File: specification/migrateprojects/resource-manager/readme.nodejs.md (hunk 2 above)
Comment (daschult) [inline specification/migrateprojects/resource-manager/readme.nodejs.md]: Please reflect your service's complete name (`azure-arm-migrateprojects`).
```suggestion
  package-name: azure-arm-migrateprojects
```

File: specification/migrateprojects/resource-manager/readme.nodejs.md (hunk 1 above)
Comment (daschult) [inline specification/migrateprojects/resource-manager/readme.nodejs.md]: Please change this to `migrateprojectsManagement`.
```suggestion
  output-folder: $(node-sdks-folder)/lib/services/migrateprojectsManagement
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by praries880 — APPROVED** [review]

**Comment by praries880** [comment]
@daschult are you satisfied with the changes?

**Review by ghost — APPROVED** [review]

**Comment by praries880** [comment]
@azuresdkci add to whitelist

**Comment by peetlotla** [comment]
Hi,

The PR was raised for ARM review. We were not looking forward to merge into master or generate the SDKs for this change as we may have modifications to the swagger based on the review. Can you please revert the merge?

Thanks,
-Peet

**Comment by peetlotla** [comment]
Pareekshit,

Can you please let us know if this is reverted?

From: Peetambaram Lotla
Sent: Friday, November 9, 2018 10:13 AM
To: Azure/azure-rest-api-specs <reply@reply.github.com>; Azure/azure-rest-api-specs <azure-rest-api-specs@noreply.github.com>; Pareekshit Singh <Pareekshit.Singh@microsoft.com>
Cc: Author <author@noreply.github.com>; Gaurav Bhatnagar <Gaurav.Bhatnagar@microsoft.com>
Subject: RE: [Azure/azure-rest-api-specs] Swagger closure for Azure Migration Hub Operations (#4330)

Hi,

The PR was raised for ARM review. We were not looking forward to merge into master or generate the SDKs for this change as we may have modifications to the swagger based on the review. Can you please revert the merge?

Thanks,
-Peet

**Pull request merged**